lib/improver/tests/utilities/test_save.py

```diff
@@ -177,8 +176,36 @@ def test_cf_shared_attributes_list(self):
         self.assertTrue(all(key in self.global_keys_ref
                             for key in global_keys_in_file))
 
+    def test_error_unknown_units(self):
```

Comment:
Why have you chosen to put this in her rather than the Test__check_for_units class?